app/components/card-sort/component.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+
+export default Ember.Component.extend({
+  actions: {
```

Comment:
It can be nice to indicate what attributes the component will have by doing: 
`<attribute>: null` 
at the top of the class definition.